### ISSUE TITLE
Add update for v4 

### ISSUE THREAD

**Issue by abhijitgujar86** [dgrijalva/jwt-go#415]
Its good to see that v4 is available on golang  packages

https://pkg.go.dev/github.com/dgrijalva/jwt-go/v4?tab=doc

but it seems there no other details. also the git link for the same is 404

**Comment by Ayanrocks** [comment]
Yes, this should b fixed. If you're releasing v4 beta at least add docs to it. It's been 8 months since it's release and I'm having trouble implementing it
